charts/rucio-server/values.yaml

```diff
@@ -146,6 +146,7 @@ wsgi:
 httpd_config:
   mpm_mode: "event"
   enable_status: "True"
+  legacy_dn: "False"
```

Comment:
This will have no effect. You have to add use the RUCIO_HTTPD_LEGACY_DN env variable correctly somewhere in https://github.com/rucio/containers/blob/985da051dc13462482dcc47f79384c7dad5bc3e0/server/httpd.conf.j2#L100